sql/2021/security/feature_adoption_by_topN_technologies.sql

```diff
@@ -0,0 +1,94 @@
+#standardSQL
+# Determines to what extent the top-N technology drivers are responsible for the global adoption of different security features
+-- from https://stackoverflow.com/a/54835472
+CREATE TEMP FUNCTION array_slice(arr ARRAY<STRING>, start INT64, finish INT64)
+RETURNS ARRAY<STRING> AS (
+  ARRAY(
+    -- SQL Linter can't handle WITH OFFSET syntax so escape it
+    SELECT part FROM UNNEST(arr) part WITH OFFSET index  -- noqa: PRS
```

Comment:
Discovered this comment was actually wrong of mine (fixed 2020 query in `main` btw):

```suggestion
    SELECT part FROM UNNEST(arr) part WITH OFFSET AS index
```